src/queries/savedStories.tsx

```diff
@@ -0,0 +1,81 @@
+import supabase from '../utils/supabase';
+
```

Comment:
I think it would be good to create a const variable for the table name in case we change it in the future. Usually, database tables follow `snake_case` so we should change the table name to `saved_stories`.